build-support/packer/build.sh

```diff
@@ -127,10 +133,10 @@ function warm_artifact_cache {
   # Fetch the mesos egg, needed to build python components.
   # The mesos.executor target in 3rdparty/python/BUILD expects to find the native egg in
   # third_party.
-  SVN_ROOT='https://svn.apache.org/repos/asf/aurora/3rdparty'
+  MESOS_PY_EGGS='https://dl.bintray.com/aurora-scheduler/python-eggs'
   pushd "$THIRD_PARTY_DIR"
     wget -c \
-      ${SVN_ROOT}/ubuntu/xenial64/python/mesos.executor-${MESOS_VERSION}-py2.7-linux-x86_64.egg
+      ${MESOS_PY_EGGS}/ubuntu/xenial64/mesos.executor-${MESOS_VERSION}-py2.7-linux-x86_64.egg
```

Comment:
I get 'Forbidden' when trying to get https://dl.bintray.com/aurora-scheduler/python-eggs/ubuntu/xenial64/mesos.executor-1.8.1-py2.7-linux-x86_64.egg
Do you know if it is due to rate limiting? Or bintray being end of life? (see https://jfrog.com/center-sunset/)

Reply:
We don't use the packer :)

Reply:
OK, so why do we have a diff here if we are not using this component? ^^

Reply:
Because I cherry-picked upstream patches directly - they made all the work already so I thought it was smarter to reuse the patches to do that, without taking other code related commits

Reply:
Got it, I was looking at the overall changes and did not see you cherry-picked upstream patches, thanks for your answer :)